.github/workflows/build-watcher-operator.yaml

```diff
@@ -0,0 +1,25 @@
+name: watcher operator image builder
+
+on:
+  push:
+    branches:
+      - '*'
+
+env:
+  imageregistry: 'quay.io'
+  imagenamespace: ${{ secrets.IMAGENAMESPACE || secrets.QUAY_USERNAME }}
+  latesttag: latest
+
+jobs:
+  call-build-workflow:
+    uses: openstack-k8s-operators/openstack-k8s-operators-ci/.github/workflows/reusable-build-operator.yaml@main
+    with:
+      operator_name: watcher
+      go_version: 1.21.x
+      operator_sdk_version: 1.31.0
+    secrets:
+      IMAGENAMESPACE: ${{ secrets.IMAGENAMESPACE }}
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+      REDHATIO_USERNAME: ${{ secrets.REDHATIO_USERNAME }}
+      REDHATIO_PASSWORD: ${{ secrets.REDHATIO_PASSWORD }}
```

Comment:
These are Organization secrets available to all repos.

Reply:
we also have https://github.com/openstack-k8s-operators/nova-operator/blob/main/.github/workflows/build-operator-dont-publish.yaml in nova

the difference is the build-operator-dont-publish.yaml runs on PRs
the where as this one runs only after the change merges since it runs on "push" to a branch which includes merging  pr

i guess since we have the meta content provider we can skip the don't push version so this looks reasonable to me

we also have the prow job to build the images in fips mode so we can probably drop the build-operator-dont-publish.yaml github action form nova now.